src/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import HomePage from './components/homePage/HomePage';
+import MyAppointments from './components/appointmentInfo/MyAppointments';
+import DoctorList from './components/doctorInfo/DoctorList';
+import DeleteDoctor from './components/doctorInfo/DeleteDoctor';
+import PatientList from './components/forSuperUser/PatientList';
+import Appointment from './components/appointmentInfo/Appointment';
+import AppointmentList from './components/forSuperUser/AppointmentList';
+import UserList from './components/forSuperUser/UserList';
+import DoctorDetails from './components/doctorInfo/DoctorDetails';
+import ModalAppoint from './components/appointmentInfo/ModalAppoint';
+
+function App() {
```

Comment:
- [ ] When I press the sign up button as a new user, this happens:

![image](https://github.com/Tracy-miranja/appoint-doctor-frontend/assets/57421823/30d924c4-34be-4a70-bdf6-85e5de4381b0)

After that, I fill the form:

![image](https://github.com/Tracy-miranja/appoint-doctor-frontend/assets/57421823/556713bc-ff1d-4a4e-89f3-9c25ba15317a)

The password I set is "03030303".

When I finish, there is no POST request to the server.

If I then try to log in:

![image](https://github.com/Tracy-miranja/appoint-doctor-frontend/assets/57421823/37275403-6f32-4c71-8d1a-599897a3a6f6)

It fails and gives me a 401.

If this works on your end, please test this with an incognito window.  If your app requires any special procedures to be run correctly, kindly add them to your readme.

I highly suggest you don't ignore this review comment, because chances are this will happen to other users, including the final evaluator and potential employers. 👍